MAINTAINERS.yml

```diff
@@ -1416,7 +1416,9 @@ nRF52 BSIM:
         - "platform: nRF52 BSIM"
 
 POSIX API layer:
-    status: odd fixes
+    status: maintained
+    maintainers:
+        - cfriedt
```

Comment:
should remove yourself from `collaborators:` now that you are in `maintainers:`.

Reply:
Good catch ;-)